builds/e2e/templates/deploy-iotuap-setup.yaml

```diff
@@ -0,0 +1,66 @@
+parameters:
+  release.label: ''
```

Comment:
for those parameter it is not used in this template, please remove them.

Reply:
Removed.